be/src/runtime/etl_job_mgr.cpp

```diff
@@ -62,6 +63,21 @@ const std::string ERROR_FILE_PREFIX = "error_log";
 
 EtlJobMgr::EtlJobMgr(ExecEnv* exec_env) :
         _exec_env(exec_env), _success_jobs(5000), _failed_jobs(5000) {
+    REGISTER_PRIVATE_VARIABLE_METRIC(running_etl_job_count);
```

Comment:
`EtlJobMgr` is deprecated, it will be removed in next version.
So you don't need to add metrics for it.